src/components/Modals/GettingStartedModal.tsx

```diff
@@ -11,10 +11,10 @@ import {
   AlertActionLink,
```

Comment:
We already have `src/components/modal`, should we move modals there instead of having two `modal` and `Modals` directories?

Reply:
I was trying to leave conflicts out of this PR. There are 2 `GettingStartedModal` components, one for app-studio, one for hacbs. When we do the work to resolve this we can combine the directories.